### ISSUE TITLE
Remove FTP support

### ISSUE THREAD

**Issue by domenic** [whatwg/fetch#1009]
It seems Chrome and Firefox are both removing FTP support.

Potentially impacted parts of the spec:

- [Scheme fetch](https://fetch.spec.whatwg.org/#scheme-fetch)
- [Network scheme](https://fetch.spec.whatwg.org/#network-scheme): maybe this can be removed in favor of "HTTP(S) scheme"? Or rename "HTTP(S) scheme" to "network scheme"? Unsure what this impacts.
- [Bad port blocking](https://fetch.spec.whatwg.org/#port-blocking) (connected to network scheme)
- [Main fetch](https://fetch.spec.whatwg.org/#main-fetch) step 2.8 appears to be attempting to network-error some FTP requests. If we instead network-error all FTP requests then this step can probably be deleted.

**Comment by bakkot** [comment]
> Network scheme: maybe this can be removed in favor of "HTTP(S) scheme"? Or rename "HTTP(S) scheme" to "network scheme"? Unsure what this impacts.

"Network scheme" is [used](https://w3c.github.io/webappsec-csp/#match-url-to-source-expression) at least in CSP, so that e.g. `script-src *` allows loading scripts hosted on `http://example.com`, `https://example.com`, and `ftp://example.com`, but does not allow `<script src="data:text/javascript,alert(1)"></script>`.

**Comment by domenic** [comment]
Makes sense. But after removing FTP from browsers, it seems CSP could just replace that with HTTP(S) scheme. (Or we could replace all uses of HTTP(S) scheme with network scheme.)

**Comment by bakkot** [comment]
Yup, that's my reading too. That change would be observable, because it will mean that `<script src="ftp://example.com">` on a page with `script-src *` will start triggering a user-observable CSP violation event, but that seems fine.

**Comment by annevk** [comment]
I'd be inclined to remove "network scheme" and only have "HTTP(S) scheme" as any new kind of protocol would require a lot of changes anyway (and has a much higher bar to clear than FTP). If anything still requires safelisting "ftp" for compatibility reasons they could do so explicitly to indicate the legacy need.
